### DISCUSSION TITLE
Branch verschwunden

### DISCUSSION THREAD

**Discussion by Linebeck0** [rjcncpt/StarCitizen-Deutsch-INI#425] [Übersetzungsvorschläge]
Hallo Zusammen,

damit ich einfach immer meine Änderungen pushen kann, hatte ich mir extra einen benamten Branch erstellt.
Diesen scheint ihr wieder entsorgt zu haben.

Dies finde ich schade, da es so die Arbeit unnötig verkompliziert.
Des weiteren möchte ich darauf hinweisen, das offensichtlich einige nicht die Gramatik der deutschen Sprache beachten und dies offensichtlich gedulded wird. Anders kann ich mir nicht erklären warum Satzzeichen als Rudeltiere commited werden können.

Ebenso wie manch andere Anregungne würde ich so etwas (!!!) vorschlagen via Github Action zu unterbinden

Ach und falls das gleich jemand beseitigen sollte, hinter Kollisionsgefahr muss überhaupt kein Ausrufezeichen ;-).

Gruß

**Comment by rjcncpt** [comment]
Hallo. Also das war nun wirklich keine Absicht. Und wir löschen branches allgemein irgendwann. Dann hättest du auch gern etwas sagen können. Es kann ja nun wirklich keiner wissen.

Btw ist der Ton von dir unterirdisch und niemand von uns muss sich als analphabeten bezeichnen lassen. Auf derartigen Umgangston können wir verzichten. 

Alles Gute!

**Comment by greluc** [comment]
#426
